CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.375 May 26 2024
+- Add `ManagementUpgrade` parameters to the `NodePool` model to support additional upgrade configuration.
```

Comment:
Please include as well
- Rename file for `access_request_status_type` to .model